src/main/java/com/ericsson/ei/frontend/WebController.java

```diff
@@ -42,6 +44,8 @@ public String greeting(Model model) {
     @RequestMapping("/subscriptionpage.html")
     public String subscription(Model model) {
         model.addAttribute("frontendServiceUrl", frontEndUtils.getFrontEndServiceUrl());
+        model.addAttribute("invalidName", RegExProviderUtils.getRegEx("invalidName"));
+        model.addAttribute("validEmail", RegExProviderUtils.getRegEx("validEmail"));
```

Comment:
Check my review comments in backend pull requests.

Reply:
Took me a while to understand your comment, but then validName could be subscriptionName and validEmail could be notificationMeta?

Although I like the suggestion, but notificationMeta could be url or email, currently we do not use regex for url, but if we add that, how would it validate both email and url?